etc/nginx/snippets/rewrites.conf

```diff
@@ -1,17 +1,17 @@
 # NOTE rewrite uses 302 redirect by default (assuming replacement does not start with a protocol)
 # current versions
-set $current_version_sdk_c_api '3.3.10';
-set $current_version_sdk_cxx_api '1.0.0-dp.10';
-set $current_version_sdk_dotnet_api '3.4.13';
-set $current_version_sdk_go_api '2.7.0';
-set $current_version_sdk_java_api '3.5.0';
-set $current_version_sdk_jvm_core_api '2.5.0';
-set $current_version_kotlin_client_api '1.2.0';
-set $current_version_sdk_nodejs_api '4.2.8';
-set $current_version_sdk_php_api '4.1.6';
-set $current_version_sdk_python_api '4.1.9';
-set $current_version_sdk_ruby_api '3.4.5';
-set $current_version_sdk_scala_api '1.5.0';
+set $current_version_sdk_c_api '3.3.12';
+set $current_version_sdk_cxx_api '1.0.0-dp.15';
```

Comment:
If these are being updated, is there any reason not to update them to latest to match prod?

Reply:
Ignore this PR, it was caused by a mixup between forks of docs-site